### DISCUSSION TITLE
Problem with opening the pause menu at game speed 200% and above

### DISCUSSION THREAD

**Discussion by Kiwi1444** [ikemen-engine/Ikemen-GO#2117] [Q&A] [answered]
Is it only me experiencing problems with opening pause menu at game speed 200% and above?
To reproduce:
1. Start the match
2. Press ctrl+S
3. Try to call pause menu (esc button)

**Comment by potsmugen** [comment, marked as answer]
The actual pause menu button isn't Esc though. It's the Menu button defined in the key config. I only get that problem with Esc.
